packages/python-runner/style/index.css

```diff
@@ -21,6 +21,7 @@
 .elyra-PythonEditor-OutputArea-child {
   border-top: 1px solid var(--jp-border-color2);
   border-bottom: 1px solid var(--jp-border-color2);
+  margin-right: 64px;
```

Comment:
this should be moved to `.elyra-PythonEditor-OutputArea-output` otherwise the bottom board can look odd
<img width="194" alt="Screen Shot 2020-03-20 at 3 07 27 PM" src="https://user-images.githubusercontent.com/13952758/77209830-98e8a200-6abc-11ea-81a1-143d14b30fd1.png">

Reply:
it also looks a bit odd with the change, but I would personally say better than without
<img width="179" alt="Screen Shot 2020-03-20 at 3 09 10 PM" src="https://user-images.githubusercontent.com/13952758/77209946-ed8c1d00-6abc-11ea-91a9-c305196b6768.png">
<img width="143" alt="Screen Shot 2020-03-20 at 3 09 18 PM" src="https://user-images.githubusercontent.com/13952758/77209947-ee24b380-6abc-11ea-8b1d-1059520953b6.png">

Reply:
Yeah I did that deliberately but I wasn't sure which looked weirder. I could also make the background white instead of transparent?

Reply:
How does this look? 
![image](https://user-images.githubusercontent.com/6673460/77210696-a3606700-6acf-11ea-9e28-61179ae511a0.png)

Reply:
I'd say it's a decent middle ground between the two not so great choices. Lets go with that